hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -181,6 +196,40 @@ Since the updates files are in the same format as generated by rpi-eeprom-update
 Default: 0  
 Version: pieeprom-2020-04-16.bin  
 
+**pieeprom-2020-05-15.bin - BETA**
+SELF_UPDATE and bootloader_update are now enabled by default so that rpi-update will work by default. Setting either of these parameters to zero prevents self-updates.
+
+### FREEZE_VERSION
+Previously this property was only checked by the rpi-eeprom-update script. However, now that self-update is enabled the bootloader will also check this field. If set, this overrides SELF_UPDATE to stop automatic updates. To disable FREEZE_VERSION you will have to use and SD card boot with recovery.bin.
```

Comment:
Typo: use and SD card boot -> use an SD card boot

Reply:
you can also use the flashrom mode in `rpi-eeprom-update` to either update it directly or change the `FREEZE_VERSION`

Reply:
flashrom mode is about to be deleted from the scripts

Reply:
done